modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/src/main/resources/META-INF/resources/js/guides/TableGuide.js

```diff
@@ -0,0 +1,458 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayTable from '@clayui/table';
+import React from 'react';
+
+import TokenGroup from '../components/TokenGroup';
+import TokenItem from '../components/TokenItem';
+
+const TABLE = [
+	{
+		className: 'table-divided',
+		hover: false,
+		responsive: false,
+	},
+	{
+		cellClassName: 'cell-spaced',
+		className: 'table-spaced',
+		hover: false,
+		responsive: false,
+	},
+	{
+		className: 'table-bordered',
+		hover: false,
+		responsive: false,
+	},
+];
+
+const DENSITY = [
+	{
+		className: 'default',
+		hover: false,
+		responsive: false,
+	},
+	{
+		className: 'table-sm',
+		hover: false,
+		responsive: false,
+	},
+	{
+		className: 'table-lg',
+		hover: false,
+		responsive: false,
+	},
+];
+
+const UTILITY_CLASSES = [
+	{
+		className: 'table-hover',
+		hover: true,
+		responsive: false,
+	},
+	{
+		className: 'table-responsive',
+		hover: false,
+		responsive: true,
+	},
+];
+
+const BIG_TEXT =
+	'Lorem ipsum dolor sit amet, con Lorem ipsum dolor sit amet, con Lorem ipsum dolor Lorem ipsum dolor sit amet, con Lorem ips Lorem';
+
+const Table = ({
+	cellClassName,
+	cellExpanded,
+	className,
+	hover,
```

Comment:
If we add falsey defaults for hover and responsive, we won't have to pass false to it everywhere.
```
hover = false,
responsive = false,
```

Reply:
Fixed